web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx

```diff
@@ -57,37 +58,23 @@ export function CliCommand({
       bg="bgTerminal"
       mb={2}
     >
-      <Flex
+      <CommandContainer
         mr="2"
         width="100%"
         shouldDisplayIsLoading={shouldDisplayIsLoading}
-        css={`
-          overflow: auto;
-          white-space: pre;
-          word-break: break-all;
-          font-size: 12px;
-          color: ${props => {
-            // always use light colors
-            const { light } = props.theme.colors;
-            // 0.72 - text.slightlyMuted opacity
-            return props.shouldDisplayIsLoading ? fade(light, 0.72) : light;
-          }};
-          font-family: ${props => props.theme.fonts.mono};
-        `}
       >
         <Box mr="1">{`$`}</Box>
         <span>{cliCommand}</span>
         {shouldDisplayIsLoading && (
           <Indicator
-            fontSize="14px"
```

Comment:
@gzdunek This component here doesn't work quite well. ;f

@bl-nero Could you change this to `size={14}` to make it less broken?

Reply:
This request is so quick that it's hard to notice that something is broken hah

Reply:
@ravicious @gzdunek It looks even worse with size={14}; I'm assuming you meant size={24} and a `line-height` of 0; please take a look and verify.

Reply:
I think we avoid adding more props to this component and instead do this:

```
Subject: [PATCH] Fix the loading indicator
---
Index: web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx b/web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx
--- a/web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx	(revision c811b34f40977a0a1201f36c04708711b83d2129)
+++ b/web/packages/teleterm/src/ui/DocumentGateway/CliCommand.tsx	(date 1718014860822)
@@ -17,7 +17,7 @@
  */
 
 import React, { useEffect, useState } from 'react';
-import { Box, ButtonPrimary, Flex, Indicator } from 'design';
+import { ButtonPrimary, Flex, Indicator } from 'design';
 import { fade } from 'design/theme/utils/colorManipulator';
 import styled from 'styled-components';
 
@@ -63,16 +63,14 @@
         width="100%"
         shouldDisplayIsLoading={shouldDisplayIsLoading}
       >
-        <Box mr="1">{`$`}</Box>
-        <span>{cliCommand}</span>
-        {shouldDisplayIsLoading && (
+        <span>$ {cliCommand}</span>
+        {!shouldDisplayIsLoading && (
           <Indicator
-            size={24}
+            color="inherit"
+            size="medium"
             delay="none"
             css={`
-              line-height: 0;
-              display: inline;
-              margin: auto 0 auto auto;
+              display: flex;
             `}
           />
         )}
@@ -99,6 +97,7 @@
   white-space: pre;
   word-break: break-all;
   font-size: 12px;
+  justify-content: space-between;
   color: ${props => {
     // always use light colors
     const { light } = props.theme.colors;

```
It makes it look like that:
<img width="588" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/7b7b099d-3645-4374-b42c-0b361c1551ef">

Reply:
This is how the new version looks like for me with scrollbars always on (since I use a mouse). But nvm, it's better than the previous version anyway and this call shouldn't take more than a few ms anyway.

https://github.com/gravitational/teleport/assets/27113/c04395fd-398b-49b3-8b12-b45eadc48ce7